src/types.db

```diff
@@ -82,6 +82,7 @@ file_size               value:GAUGE:0:U
 files                   value:GAUGE:0:U
 flow                    value:GAUGE:0:U
 fork_rate               value:DERIVE:0:U
+free_hugepages          value:DERIVE:0:U
```

Comment:
If I understand [hugetlbpage.txt](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) correctly, then the correct [data source type](https://collectd.org/wiki/index.php/Data_source) for these metrics would actually be `GAUGE`, i.e. the reported numbers are *not* monotonically increasing. The metrics reported by this plugin should use one "type" and use "type instance" to differentiate between the metrics. In that case, I would recommend to use the existing `vmpage_number` to report these metrics.

*   vmpage_number-free
*   vmpage_number-reserved
*   vmpage_number-surplus
*   …

Since the size of hugepages appears to be a setting, please consider to provide the user with the option to convert the metrics to bytes. In that case, the `bytes` "type" should be used.